.travis.yml

```diff
@@ -29,11 +26,15 @@ script:
 matrix:
     fast_finish: true
     include:
-      - php: 5.6
-        env: SYMFONY_VERSION='~2.7.0'
-      - php: 5.6
-        env: SYMFONY_VERSION='~2.8.0'
-      - php: 5.6
-        env: SYMFONY_VERSION='~3.4.0'
+      - php: 7.2
+        env: SYMFONY_VERSION='~4.1.0'
+      - php: 7.2
+        env: SYMFONY_VERSION='~4.2.0'
```

Comment:
No need to test 4.1 and 4.2, they are no longer maintained. Just to custom stage with `SYMFONY_VERSION=^4.0` & `SYMFONY_VERSION=^5.0`